src/main/resources/en_US.properties

```diff
@@ -44,4 +44,5 @@ message_cls_notice=Dragonet CLS system enabled. \n\u00a7eYou can login at \u00a7
 message_server_error=\u00a7cServer was unable to log you in because it had an error! \n\u00a7cPlease try join again later... :( \n\u00a70.\n\u00a7cReason: %s
 error_cls_unreachable=Unable to reach CLS HTTP API
 error_cls_error=CLS System replied weird stuff. Please retry.
-error_cls_login=Faild to login with your login token, try re-login on CLS? 
+error_cls_login=Faild to login with your login token, try re-login on CLS?
```

Comment:
Isn't it Failed instead of Faild?

Reply:
@DefinitelyEvil has always said faild, idk why

Reply:
meh that's my bad... xD (so embarrasing)